src/nvidia-cuda/install.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+set -e
+
+INSTALL_CUDNN=${INSTALLCUDNN:-"false"}
+INSTALL_NVTX=${INSTALLNVTX:-"false"}
+CUDA_VERSION=${VERSION:-"latest"}
+CUDNN_VERSION=${CUDNNVERSION:-"latest"}
+
+# NVIDIA's package names include this information
+LATEST_CUDA_VERSION="11.7"
+LATEST_CUDNN_VERSION="8.5.0.96-1"
```

Comment:
Discussed offline about moving these values to the `devcontainer-feature.json`. 🚀